core/.changelog.d/2293.added

```diff
@@ -0,0 +1 @@
+Success animation for Model R
```

Comment:
nothing simpler than deleting it, but in the past i was specifically asked to add log entry for model R by @matejcik - see https://github.com/trezor/trezor-firmware/pull/2243#issuecomment-1121003770 - this seems to be contradicting it

Reply:
I agree with removal here. I wouldn't say "only changes that affect current users", developers are also an audience for the changelog, but it is a matter of granularity. "We had Model R support since here" is good to know; "we had success animation", not so much.

Reply:
removed https://github.com/trezor/trezor-firmware/commit/aa9b5b24f01fb561460af15143fa54153056abe4